api/schema_test.go

```diff
@@ -0,0 +1,65 @@
+package api
```

Comment:
also add functest pls

Reply:
It is a serious question, do we need to functest this. 
I belive this mechanism will be obsoleted in next version of platform.
So, if we do not want to develop it, may be we have no need to invest in testing?

Reply:
how do you prove that it works without tests? 
we can remove or not. we'll make a decision later. I really think that we have to develop a test.